client/doc/onboarding.md

```diff
@@ -0,0 +1,64 @@
+## Onboarding component
```

Comment:
To my mind, this is the description/doc of the onboarding machine, not the doc of the component that will be handled by this onboarding object (which is your PR).